contact/index.html

```diff
@@ -177,7 +179,9 @@ <h2 class="content-page-title">CONTACT / お問い合わせ</h2>
       <p class="global-footer__copy">
         <small>All content Copyright ニッカオフィス / knicka office</small>
       </p>
+    </main>
     </futter>
     <script src="/contact/contact-script.js"></script>
+    </div>
```

Comment:
```suggestion
    </div>
    <script src="/contact/contact-script.js"></script>
```
`script` 要素は、`</body>` の直前に置きます。